cartographer_ros/configuration_files/assets_writer_backpack_2d.lua

```diff
@@ -19,20 +19,7 @@
 -- of a pixel in a X-Ray.
 VOXEL_SIZE = 5e-2
 
-XY_TRANSFORM =  {
-  translation = { 0., 0., 0. },
-  rotation = { 0., -math.pi / 2., 0., },
-}
-
-XZ_TRANSFORM =  {
-  translation = { 0., 0., 0. },
-  rotation = { 0. , 0., -math.pi / 2, },
-}
-
-YZ_TRANSFORM =  {
-  translation = { 0., 0., 0. },
-  rotation = { 0. , 0., math.pi, },
-}
+include "transform.lua"
```

Comment:
include also in the other lua files that define these transforms.

Reply:
DOne.